tasks/managed/rh-sign-image/rh-sign-image.yaml

```diff
@@ -113,14 +113,12 @@ spec:
         REQUESTTYPE=$(jq -r '.sign.requestType // "internal-request"' "${DATA_FILE}")
         service_account_name=$(jq -r '.spec.pipeline.serviceAccountName // "appstudio-pipeline"' "${RPA_FILE}")
         if [ "${REQUESTTYPE}" == "internal-pipelinerun" ] ; then
-          requestType=internal-pipelinerun
+          requestType=(internal-pipelinerun -r)
```

Comment:
Hmm, would it make sense to modify the `internal-pipelinerun` script to also use --pipeline? I think the only reason it's called request there is for consistency with the `internal-request` script. But in reality it aligns better with the --pipeline parameter.

And besides, it looks a bit cumbersome to still call this `requestType` if it's gonna be an array like this. If you go with the script change instead, you wouldn't need this.

Reply:
yeah I think it makes sense. I can update the utils script